File: EpicWorkout/EpicWorkout/EpicWorkout.Android/Fragments/ExerciseItems/ExerciseViewHolder.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using Android.Support.V7.Widget;

namespace EpicWorkout.Droid.Fragments.ExerciseItems
{
    public class ExerciseViewHolder : RecyclerView.ViewHolder
    {
        public ImageView exerciseImageView { get; set; }
        public TextView descriptionTextView { get; set; }
        public TextView nameTextView { get; set; }
        public View itemView { get; set; }

        public ExerciseViewHolder(View itemView) : base(itemView)
        {
            exerciseImageView = itemView.FindViewById<ImageView>(Resource.Id.thumbnail);
            descriptionTextView = itemView.FindViewById<TextView>(Resource.Id.description);
            nameTextView = itemView.FindViewById<TextView>(Resource.Id.name);
            this.itemView = itemView;
        }

        public void setName(String name)
        {
            nameTextView.Text = name;
        }
        public void setDescription(String description)
        {
            descriptionTextView.Text = description;
        }
    }
}

File: EpicWorkout(Android Studio)/app/src/main/java/com/epicworkout/fernaak/epicworkout/ExerciseItems/ExerciseViewHolder.java
package com.epicworkout.fernaak.epicworkout.ExerciseItems;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.epicworkout.fernaak.epicworkout.R;

public class ExerciseViewHolder extends RecyclerView.ViewHolder {
    private ImageView exerciseImageView;
    private TextView descriptionTextView;
    private TextView nameTextView;
    public View itemView;

    public ExerciseViewHolder(View itemView) {
        super(itemView);
        exerciseImageView = itemView.findViewById(R.id.thumbnail);
        descriptionTextView = itemView.findViewById(R.id.description);
        nameTextView = itemView.findViewById(R.id.name);
        this.itemView = itemView;
    }

    public void setName(String name) {
        nameTextView.setText(name);
    }
    public void setDescription(String description) {
        descriptionTextView.setText(description);
    }
    public void setExerciseImageView(){ exerciseImageView.setImageResource(R.drawable.barbell2);}
}
